hw/ip/pattgen/data/pattgen.prj.hjson

```diff
@@ -0,0 +1,21 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{
+    name:               "pattern",
+    design_spec:        "hw/ip/pattern/doc",
+    dv_plan:            "hw/ip/pattern/doc/dv_plan",
+    hw_checklist:       "hw/ip/pattern/doc/checklist",
+    revisions: [
+      {
+        version:            "0.5",
+        life_stage:         "L1",
+        design_stage:       "D1",
+        verification_stage: "V1",
```

Comment:
Given this commit only adds the documentation, this should all be V0/D0 probably, and can then be incremented in the next commits.

@sriyerg @sjgitty  to confirm.

Reply:
Oh, no problem, thanks for pointing that out.  Normally I would have left this out of the initial PR entirely, but there are a number of dependencies now with the hardware dashboard.
You are right though V1/D1 is a *definite* error at this point. Thanks for catching.

Reply:
Corrected in upcoming commit.  I suspect someone just copied this from another IP.   I will also correct the misspelling of "pattgen" here.